### ISSUE TITLE
Nested category facet button is confusing

### ISSUE THREAD

**Issue by jacobthill** [sul-dlss/dlme#1347]
The window icons for the Type facet are clickable (clicking yields the sub categories) but this affordance was not clear to any of the users we interviewed. We need something in the UI to make it clear that the buttons are clickable.
<img width="234" alt="Screen Shot 2021-11-08 at 11 42 21 AM" src="https://user-images.githubusercontent.com/37662787/140782994-7dbde226-a10a-4dd2-bc0f-0d5719fd5612.png">

**Comment by ggeisler** [comment]
@jacobthill Yes, we definitely need to replace that icon. What do you think about using these, which are FontAwesome icons and should be easy for us to use (in theory, at least. This is a separate gem -- blacklight-hierarchy -- and I don't know what is involved in overriding the icon, but it seems like we should be able to):

### Closed
<img width="528" alt="Screen Shot 2021-11-08 at 3 34 04 PM" src="https://user-images.githubusercontent.com/101482/140830250-726d28db-ea8c-464f-be53-bca06920aee5.png">

### Open
<img width="528" alt="Screen Shot 2021-11-08 at 3 35 49 PM" src="https://user-images.githubusercontent.com/101482/140830296-ff98fa18-fde1-45e3-bb4c-7c7decbe4edd.png">

**Comment by jacobthill** [comment]
Those look much better. I'm removing the design needed label. To clarify, are you suggesting this should be implemented in DLME only or implemented in Blacklight hierarchy?

**Comment by ggeisler** [comment]
If it's technically feasible, I'd suggest we do it in [blacklight-hierarchy](https://github.com/sul-dlss/blacklight-hierarchy), since we have the same issue in Searchworks:

<img width="383" alt="Screen Shot 2021-12-06 at 10 27 57 AM" src="https://user-images.githubusercontent.com/101482/144893227-c52a5c37-987c-4b2b-bc9f-1107d76a0fa6.png">

It looks like the gem intentionally [includes a way to easily override the icons](https://github.com/sul-dlss/blacklight-hierarchy#overriding-the-icon) in the consuming application, but since the default icons are less than ideal, updating the defaults to be less confusing seems like a better way to go. 

There might be technical considerations I'm not aware of, though, so I think we can rely on the developer working on this to figure out the best solution (i.e., whether to update the blacklight-hierarchy default icons or just override them in DLME).
